edgeml_pytorch/graph/rnn.py

```diff
@@ -332,14 +339,20 @@ def __init__(self, input_size, hidden_size,
                  name="FastRNN"):
         super(FastGRNNCell, self).__init__(input_size, hidden_size,
```

Comment:
Why is this FastGRNNCell and not FastRNNCell in the super()

Reply:
fixed it